tests/test_wasm_intrinsics_simd.c

```diff
@@ -471,19 +471,19 @@ v128_t TESTFN i16x8_sub_saturate_u(v128_t x, v128_t y) {
   return wasm_u16x8_sub_saturate(x, y);
 }
 v128_t TESTFN i16x8_min_s(v128_t x, v128_t y) {
-  return wasm_i16x8_min_s(x, y);
+  return wasm_i16x8_min(x, y);
 }
 v128_t TESTFN i16x8_min_u(v128_t x, v128_t y) {
-  return wasm_i16x8_min_u(x, y);
+  return wasm_u16x8_min(x, y);
 }
 v128_t TESTFN i16x8_max_s(v128_t x, v128_t y) {
-  return wasm_i16x8_max_s(x, y);
+  return wasm_i16x8_max(x, y);
 }
 v128_t TESTFN i16x8_max_u(v128_t x, v128_t y) {
-  return wasm_i16x8_max_u(x, y);
+  return wasm_u16x8_max(x, y);
 }
 v128_t TESTFN i16x8_avgr_u(v128_t x, v128_t y) {
```

Comment:
btw why foes `avgr` only have a `u` variant?

Reply:
I think it's because a signed variant could not be efficiently implemented on all architectures.